exonum-java-binding/core/rust/integration_tests/tests/service_runtime.rs

```diff
@@ -32,6 +32,7 @@ lazy_static! {
     static ref VM: Arc<JavaVM> = create_vm_for_tests_with_fake_classes();
 }
 
+#[ignore]
```

Comment:
Why are they ignored?

Reply:
Because Testkit attempts to create a block and this fails on broken proofs code (state hash). Added comment.